modules/monitoring-deploying-a-sample-service.adoc

```diff
@@ -8,6 +8,10 @@
 
 To test monitoring of a service in a user-defined project, you can deploy a sample service.
 
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role or as a user with administrative permissions for the namespace.
```

Comment:
As I see it, monitoring docs shouldn't be talking about RBAC/permissions that aren't created/managed by the monitoring stack to avoid providing any incomplete/out of date info.
If users are having a permission issue while creating a namespace or a deployment, the relevant OCP docs should help with that, if we want to help finding that doc, we can add "additional resources" with links to it.
(I see we're talking about such permissions that we don't control in other places and I don't think we should be doing that)

Reply:
I am not sure about this one, because it is necessary for procedures to contain required prerequisites, even if they are not necessarily the ones we control, because we are still a part of OpenShift. Even if the prerequisites were to change. I think removing such prerequisites would leave a big gap in our docs :sweat_smile: